pkg/system/system.go

```diff
@@ -385,7 +385,10 @@ func RunStatus(cmd *cobra.Command, args []string) {
 		log.Println("")
 	} else {
 		log.Printf("❌ System Phase is %q\n", r.NooBaa.Status.Phase)
-		CheckWaitingFor(r.NooBaa)
+		err := CheckWaitingFor(r.NooBaa)
+		if err != nil {
+			util.Logger().Fatalf("❌ %s", err)
```

Comment:
This print is not redundant because CheckWaitingFor is already printing in all of its cases - that's the reason we wrote it - to print.
So I would just use `util.IgnoreError(CheckWaitingFor(r.NooBaa))`.